Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -3558,19 +3634,25 @@ def panorama_list_rules_command(args: dict):
     if disabled := args.get('disabled'):
         rules = filter_rules_by_status(disabled, rules)
 
-    pretty_rules = prettify_rules(rules, target)
-
+    pretty_rules, context_rules = prettify_rules(rules, target)
+    
+    
     return_results({
         'Type': entryTypes['note'],
         'ContentsFormat': formats['json'],
         'Contents': rules,
         'ReadableContentsFormat': formats['markdown'],
         'HumanReadable': tableToMarkdown('Security Rules:', pretty_rules,
-                                         ['Name', 'Location', 'Action', 'From', 'To',
-                                          'CustomUrlCategory', 'Service', 'Tags', 'Disabled'],
+                                         ['Name'] + ['Location']*is_panorama + ['Tags','Type',
```

Comment:
why take out location if not panorama? If its a matter of not having null columns removeNull should do this job

Reply:
I wanted the readable outputs to mirror their respective UIs
should i just have one for both?